css-four-column-layout/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>Four Column Layout</title>
+  <link href="https://fonts.googleapis.com/css2?family=Barlow:wght@100;400&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+  <header>
+    <h1>How It Works</h1>
+  </header>
+
+  <div class="row">
+    <img src="images/screen-1.png" class="phone">
+    <img src="images/screen-2.png" class="phone">
+    <img src="images/screen-3.png" class="phone">
+    <img src="images/screen-4.png" class="phone">
+  </div>
+
+  <div class="row">
+    <img src="images/switch-icon.png" class="icon">
+    <img src="images/filter-icon.png" class="icon">
+    <img src="images/slider-icon.png" class="icon">
+    <img src="images/bars-icon.png" class="icon">
+  </div>
+
+  <div class="row">
+    <div class="feature">
+      <h3>App Switching</h3>
+      <p class="description">Toggles all your apps on and off.</p>
+    </div>
+    <div class="feature">
+      <h3>Trip Filtering</h3>
+      <p class="description">Evaluates your trip requests.</p>
+    </div>
+    <div class="feature">
+      <h3>Service Filters</h3>
+      <p class="description">Accepts only your ideal trips.</p>
+    </div>
+    <div class="feature">
+      <h3>Stats + Mileage</h3>
+      <p class="description">Track your trips acros all apps.</p>
+    </div>
+  </div>
```

Comment:
Try organizing everything into four columns that each include the phone image, icon, and feature.